terraform/modules/monitoring/variables.tf

```diff
@@ -100,3 +100,23 @@ variable "log_level" {
   description = "Log level for writting logs"
   default     = "DEBUG"
 }
+
+variable "pmap_subscription_ids" {
+  type        = list(string)
+  default     = []
+  description = "The subscription ids used in pmap"
+}
+
+## default is 3600s(1 hr)
+variable "oldest_unacked_messages_age_threshold" {
```

Comment:
Should we point out the unit is in second in name and description?

Reply:
done